docs/release_notes/release_notes.md

```diff
@@ -11,8 +11,67 @@ has_children: true
 
 ## Version 4.9.0 (July 2024)
 
+* Added support for Nessus:Pro (Nessus Professional). Check the upgrade guide to onboard the logs.
+
+* Added checkbox on alert page to prevent the alert state from being updated from the Cyences setup page. Check the upgrade guide for more details.
+
+* Added the following panels for user login activity on the **Windows** dashboard:
+    * Successful Login Events
+    * Failed Login Attempts by User
+
+* Added the following panels to the **Vulnerability** dahboard:
+    * Total Vulnerability Count By Severity
+    * New Total Vulnerability Found Over Time
+
+* Added a separate alert for password expired events called **AD - Login Failure due to Password Expired** and filtered out the password expired events from authentication alerts.
+
+* Added a section to configure the digest email configs on **Cyences Settings > Cyences App Configuration > Cyences Email Action Configuration**.
+
+* Removed the following alerts as it contains static lookup which is not maintainable and causes false positives:
+    * Email - Suspicious Subject or Attachment
+    * Email - With Known Abuse Web Service Link
+    * Ransomware - Endpoint Compromise - Malicious Package Found
+
+
+* ### Enhancements
+
+    * #### Sophos Endpoint data
+        * Removed the Sophos UI configuration from Cyences configuration page & sophosinstancedetails custom command to fetch sophos endpoints.
+        * Sophos device inventory alert will now support the 'sophos_endpoints' source. Check upgrade guide to collect the sophos endpoint logs.
+
+    * #### Device Inventory dashboard
+        * On **Possible Merge UUIDs (Devices) in Device Inventory** panel, added possible combinations of devices that can be merged based on hostname matching.
+        * Added forwarder type information for the splunk devices to the **Device Inventory** table.
+        * Added more details of the splunk devices like os, host, version, forwarder type to compare with other devices on **Possible Merge UUIDs (Devices) in Device Inventory** panel.
```

Comment:
Added more details **from** the splunk devices

Reply:
Didn't we add Splunk details also on main Device Inventory table? If not we need add that for sure.

If we have then update the point to:
* Added more details from the Splunk devices like os, host, version, forwarder-type.